.github/workflows/release.yml

```diff
@@ -0,0 +1,66 @@
+name: VIVIDUS Studio Release
+
+ on:
+   workflow_dispatch:
+     inputs:
+       release-version:
+         required: true
+         type: string
+         description: The release version
+
+ jobs:
+   release:
+     runs-on: ubuntu-latest
+
+     steps:
+       - name: Checkout repository
+         uses: actions/checkout@v3
+         with:
+           token: ${{ secrets.GH_RELEASE_TOKEN }}
+
+       - name: Install JDK 17
+         uses: actions/setup-java@v3.9.0
+         with:
+           distribution: 'temurin'
+           java-version: 17
+           cache: 'maven'
+
+       - name: Create release commit
+         run: |
+           git config --global user.email "uarlouski@gmail.com"
+           git config --global user.name "Uladzislau Arlouski"
+
+           npm version ${{ inputs.release-version }} -m "Release %s"
+
+           sed -i 's/Unreleased/${{ inputs.release-version }}/g' CHANGELOG.md
+           git add CHANGELOG.md
+           git commit --amend --date=now --no-edit
+
+       - name: Build server
+         run: |
+           export JAVA_TOOL_OPTIONS=--add-opens=java.base/java.util.concurrent=ALL-UNNAMED
+           cd vividus-studio-server && ./mvnw clean install -DskipTests
```

Comment:
```suggestion
           cd vividus-studio-server
           ./mvnw clean install -DskipTests
```

Reply:
just to save execution time, the tests are guaranteed to be successful at this point